content/sensu-enterprise/2.6/integrations/influxdb.md

```diff
@@ -31,7 +31,7 @@ annotation columns  |
 `status`            | check status
 `client`            | client name
 `action`            | action of the event
-`description`       | output from the check
+`description`       | output from the check (can be overwritten by the [notification][6] specified in the check result)
```

Comment:
"can be" feels too imprecise.There's an order of precedence followed when generating event summaries:

1. check.notification
2. check.description
3. check.output

If check.notification is present it will be used, regardless of the presence of check.description or check.output
If check.description is present it will be used instead of check.output
As a required element of a check result, check.output will always be available as a fallback

This logic applies to multiple integrations in Sensu Enterprise; perhaps all of them.